pkg/apis/flows/v1/sequence_lifecycle_test.go

```diff
@@ -324,70 +315,52 @@ func TestSequencePropagateChannelStatuses(t *testing.T) {
 
 func TestSequenceReady(t *testing.T) {
```

Comment:
In this unit tests (or another one?) could we add a test that verifies that the sequence correctly sets its audiences based off of the subscription audiences?

Reply:
Similar to https://github.com/knative/eventing/pull/7902#discussion_r1598840077